Tiltfile

```diff
@@ -103,10 +105,16 @@ def load_provider_tiltfiles():
     for repo in provider_repos:
         file = repo + "/tilt-provider.json"
         provider_details = read_json(file, default = {})
-        provider_name = provider_details["name"]
-        provider_config = provider_details["config"]
-        provider_config["context"] = repo
-        providers[provider_name] = provider_config
+        if "name" in provider_details:
```

Comment:
what about using `if type(provider_details) != type([])` (so we are a little bit more explicit about the intent)?

Reply:
Much better, thanks @fabriziopandini